### ISSUE TITLE
[UWP][Accessibility] Screen reader doesn't announce error message under 'Solitaire.JSON' middle pane.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3876]
# Platform
* UWP

# Version of SDK
master

# Details
Repro Steps:​
1. Launch the application.​
2. New screen starts appearing. Navigate to Left pane JSOn files.​
3. Now select "Solitaire.JSON" button. It will update the Middle and Right Pane. Navigate all the elements of both the panes.​
4. Navigate to middle pane and provide invalid input on editor pane.
5. Verify that screen reader is reading error message or not.

Actual Result:
Screen reader is not reading error message after providing invalid input on middle editor pane under 'Solitaire.JSON' middle pane, due to this screen reader user won't be able to know about error message information which appears over the screen.

Expected Result:
Screen reader should be read error messag(Invalid property identifier character: *. Path 'body[3].type', line 55, position 3.) after providing invalid input on middle editor pane under 'Solitaire.JSON' middle pane.
